client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -101,6 +107,9 @@ export default function LicensePreview( {
 		}
 	}, [] );
 
+	const isAtomic = site?.is_wpcom_atomic;
+	const isSiteAtomic = isEnabled( 'jetpack/pro-dashboard-wpcom-atomic-hosting' ) && isAtomic;
```

Comment:
Non-blocking:

If we're only using `isAtomic` in one place, I think using its source value directly inline still provides a reasonable level of readability:

```suggestion
	const isSiteAtomic = isEnabled( 'jetpack/pro-dashboard-wpcom-atomic-hosting' ) && site?.is_wpcom_atomic;
```